search-books/src/main/java/nix/education/util/FacetFields.java

```diff
@@ -0,0 +1,10 @@
+package nix.education.util;
+
+public class FacetFields {
+    public static final String FACET_LABEL = "label";
+    public static final String FACET_COUNT = "count";
+    public static final String FQ = "fq";
+    public static final String FROM = "from";
+    public static final String TO = "to";
+    public static final int DATE_RANGE_FACET_GAP = 5;
```

Comment:
Some of the above constants are presented here `org.apache.solr.common.params.CommonParams`

Reply:
I combined all constants in SearchConstants interface